Mod/Villains/Leviathan/Cards/ImpossibleStrengthController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Handelabra.Sentinels.Engine.Controller;
+using Handelabra.Sentinels.Engine.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Jp.ParahumansOfTheWormverse.Leviathan
+{
+    public class ImpossibleStrengthController : CardController
+    {
+        public ImpossibleStrengthController(Card card, TurnTakerController controller) : base(card, controller)
+        { }
+        
+        public override void AddTriggers()
+        {
+            // Increase damage dealt by villain cards by 1
+            AddIncreaseDamageTrigger((DealDamageAction dda) => dda.DamageSource != null && IsVillainTarget(dda.DamageSource.Card), 1);
```

Comment:
I think this should check the card's villain-ness rather than checking if it's a villain target; nontarget villain cards should get increased too (like Water Shadow)